src/cache.rs

```diff
@@ -802,14 +802,22 @@ impl Cache {
         // 1. The stdlib is meant to stay relatively light.
         // 2. Typechecking the standard library ought to occur only during development. Once the
         //    stdlib is stable, we won't have typecheck it at every execution.
+        let global_env = self.mk_types_env().map_err(|err| match err {
+            CacheError::NotParsed => CacheError::NotParsed,
+            CacheError::Error(_) => unreachable!(),
+        })?;
+        self.typecheck_stdlib_(&global_env)
+    }
+
+    /// Internal function to typecheck stdlib. Has to be public because it's used in benches.
```

Comment:
Nitpick: if it's public, it's not totally internal :stuck_out_tongue: so maybe a short description could be added, like "Typecheck the stdlib, provided the initial typing environment."